apps/multiplatform/common/src/desktopMain/kotlin/chat/simplex/common/DesktopApp.kt

```diff
@@ -31,10 +30,31 @@ import java.io.File
 val simplexWindowState = SimplexWindowState()
 
 fun showApp() = application {
+  val state = getStoredWindowState() // creates file if not exists; comes with proper defaults
```

Comment:
Better to place a comment on top of the code line to make it more visible and consistent with other comments in codebase